draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -51,7 +51,7 @@ rules to express Verifiable Credentials with JSON payloads with and without sele
 
 In the so-called Issuer-Holder-Verifier Model, Issuers issue so-called Verifiable Credentials to a
 Holder, who can then present the Verifiable Credentials to Verifiers. Verifiable
-Credentials are cryptographically signed statements about a Subject, typically the Holder.
+Credentials are cryptographically secured statements about a Subject, typically the Holder. A Holder can present these statements to a Verifier in a repudiable or non-repudiable way.
```

Comment:
I'm not sure if we can just drop those words here and expect people to understand what this means.